src/components/HorizontalDivider/HorizontalDivider.tsx

```diff
@@ -0,0 +1,15 @@
+import { HorizontalDivider as HorizontalDividerSDK } from "@deskpro/app-sdk";
+type Props = {
+  full?: boolean;
+};
+export const HorizontalDivider = ({ full }: Props) => {
+  return (
+    <HorizontalDividerSDK
+      style={{
+        width: "100vw",
+        color: "#EFF0F0",
+        marginLeft: full ? "-10px" : "0px",
```

Comment:
why -10px if the `<body>` have margins of 8px on both sides? This is a potential bug with horizontal scrolling.

Reply:
Ilia, youve always commented this everytime I used this.
It doesnt cause horizontal scrolling. I use this to start the line at the very beginning.